include/deal.II/matrix_free/cuda_matrix_free.h

```diff
@@ -78,8 +78,7 @@ namespace CUDAWrappers
   {
   public:
     using jacobian_type = Tensor<2, dim, Tensor<1, dim, Number>>;
-    // TODO this should really be a CUDAWrappers::Point
-    using point_type = Tensor<1, dim, Number>;
+    using point_type    = Point<dim, Number>;
```

Comment:
Can you grep for another `FIXME` or `TODO` about `CUDAWrappers::Point`, I think that there is another one about it.

Reply:
I can't find anything.

Reply:
Alright thanks